cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -840,26 +1372,26 @@ metadata:
     operator: rook
     storage-backend: ceph
 rules:
-- apiGroups:
-  - ""
-  - apps
-  - extensions
-  resources:
-  - secrets
-  - pods
-  - pods/log
-  - services
-  - configmaps
-  - deployments
-  - daemonsets
-  verbs:
-  - get
-  - list
-  - watch
-  - patch
-  - create
-  - update
-  - delete
+  - apiGroups:
```

Comment:
The spacing changed on everything in common.yaml. Can you revert those changes that weren't from the v1extensions conversion?

Reply:
Done.